tcms/core/middleware.py

```diff
@@ -32,10 +40,13 @@ def process_request(self, request):
                     }
                 ),
             )
+        return None
```

Comment:
A simple `return` will do. If you have a naked return above then I believe pylint will not complain about the missing return at the end. But when you have `return <something>` then pylint wants all return statements to have an argument.

Reply:
Pylint complains because of line 25: return HttpResponseRedirect(reverse("setup"))